docs/installation/configuration/env_config.rst

```diff
@@ -4,145 +4,101 @@
 Environment configuration reference
 ===================================
 
+
 Open Notificaties can be ran both as a Docker container or directly on a VPS or
 dedicated server. It relies on other services, such as database and cache
 backends, which can be configured through environment variables.
 
+
 Available environment variables
 ===============================
 
---------
+
 Required
 --------
 
-* ``DJANGO_SETTINGS_MODULE``: which environment settings to use. Available options:
-    * ``nrc.conf.production``
-    * ``nrc.conf.staging``
-    * ``nrc.conf.docker``
-    * ``nrc.conf.dev``
-    * ``nrc.conf.ci``
-
-* ``SECRET_KEY``: secret key that's used for certain cryptographic utilities. You
-  should generate one via `miniwebtool`_
+* ``SECRET_KEY``: Secret key that's used for certain cryptographic utilities. You should generate one via `miniwebtool <https://www.miniwebtool.com/django-secret-key-generator>`_.
+* ``ALLOWED_HOSTS``: a comma separated (without spaces!) list of domains that serve the installation. Used to protect against Host header attacks. Defaults to: ``(empty string)``.
 
-* ``ALLOWED_HOSTS``: a comma separated (without spaces!) list of domains that
-  serve the installation. Used to protect against ``Host`` header attacks.
 
-**Docker**
-
-Additionally, the following optional envvars MUST be set/changed when running
-on Docker, since ``localhost`` is contained within the container:
-
-* ``CACHE_DEFAULT``
-* ``CACHE_AXES``
-* ``EMAIL_HOST``
```

Comment:
I should probably look into a way to restore this

Reply:
I added these variables to the `required` group and explained that they are required for docker in the help text: https://github.com/maykinmedia/open-api-framework/pull/62